jung-visualization/src/main/java/edu/uci/ics/jung/visualization/transform/LayoutLensSupport.java

```diff
@@ -30,11 +30,19 @@
   private static final Logger log = LoggerFactory.getLogger(LayoutLensSupport.class);
   protected NetworkElementAccessor<V, E> pickSupport;
 
-  public LayoutLensSupport(VisualizationViewer<V, E> vv) {
+  public LayoutLensSupport(VisualizationViewer<V, E> vv, Dimension d) {
```

Comment:
As far as I can tell, you never need (and probably never want) to specify a Dimension that is incompatible with the dimension of the LayoutModel held by the VisualizationViewer, so you should avoid passing in a separate Dimension and just extract the Dimension from the VisualizationViewer directly.

Reply:
The layoutModel could be an AggregateLayoutModel which holds LayoutModels of multiple sizes.

Reply:
Thanks for spotting this wart. I pushed a change.